### PR TITLE
adding few more CLONE_* constant for Linux/Android.

### LINKED ISSUE #3033: Various Linux CLONE_* flags are missing
The complete list of the flags can be seen e.g. here:

https://elixir.bootlin.com/linux/v6.0.12/source/include/uapi/linux/sched.h#L11

The ones that are missing from libc at the moment seem to be:

```rust
pub const CLONE_PIDFD: ::c_int = 0x1000;
pub const CLONE_CLEAR_SIGHAND: ::c_int = 0x100000000;
pub const CLONE_INTO_CGROUP: ::c_int = 0x200000000;
pub const CLONE_NEWTIME: ::c_int = 0x80;
```

I believe they should be added to this list:

https://github.com/rust-lang/libc/blob/855a6fc48ab1634ef01836d6493d08b119bdd424/src/unix/linux_like/mod.rs#L1117

Sadly I've got my hands full at the moment, so I can't exactly patch it and see if it works, but as this is pretty minor and seems easy to fix, I thought I'd report it anyway.

### PR DESCRIPTION
**Pull request by devnexen**
closes #3033.

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thank you! @bors r+

**Comment by JohnTitor** [comment]
Seems the semver check failed on musl.

**Comment by JohnTitor** [comment]
This fails on i686-android:
```
  cargo:warning=/checkout/target/i686-linux-android/debug/build/libc-test-1798aac9b2677a53/out/main.c:33516:63: error: use of undeclared identifier 'CLONE_NEWTIME'
  cargo:warning=            static const int __test_const_CLONE_NEWTIME_val = CLONE_NEWTIME;
  cargo:warning=                                                              ^
  cargo:warning=/checkout/target/i686-linux-android/debug/build/libc-test-1798aac9b2677a53/out/main.c:33528:76: error: use of undeclared identifier 'CLONE_CLEAR_SIGHAND'
  cargo:warning=            static const  long long __test_const_CLONE_CLEAR_SIGHAND_val = CLONE_CLEAR_SIGHAND;
  cargo:warning=                                                                           ^
  cargo:warning=/checkout/target/i686-linux-android/debug/build/libc-test-1798aac9b2677a53/out/main.c:33534:74: error: use of undeclared identifier 'CLONE_INTO_CGROUP'
  cargo:warning=            static const  long long __test_const_CLONE_INTO_CGROUP_val = CLONE_INTO_CGROUP;
```

@rustbot author

**Comment by JohnTitor** [comment]
Could you resolve the merge conflict? It'd also be great if you could update the label once it's resolved.

**Comment by devnexen** [comment]
How may I change the label ? I may have forgotten.

**Comment by JohnTitor** [comment]
You can use this command:
@rustbot ready

**Comment by Mark-Simulacrum** [comment]
Closing & reopening to fix bors state synchronization, which was causing queues to be blocked across multiple repositories.

File: src/unix/linux_like/linux/gnu/mod.rs [hunk 1]
```diff
@@ -946,6 +946,8 @@ pub const ELFOSABI_ARM_AEABI: u8 = 64;
 
 // linux/sched.h
 pub const CLONE_NEWTIME: ::c_int = 0x80;
+pub const CLONE_CLEAR_SIGHAND: ::c_longlong = 0x100000000;
+pub const CLONE_INTO_CGROUP: ::c_longlong = 0x200000000;
```
Comment (JohnTitor) [inline src/unix/linux_like/linux/gnu/mod.rs:950]: I think the type difference with CLONE_NEWTIME bothers users, doesn't it?

**Pull request merged**